ui/src/components/SampleView/SampleView.css

```diff
@@ -89,14 +89,6 @@
   position: relative;
 }
 
-.btn:focus {
-  outline: none !important;
```

Comment:
This was a duplicate global style. In the remaining style declaration, I add `:not([data-default-styles])` to allow opting out (see further down).